edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
+  border: none;
+  background-image: none;
+  text-decoration: none;
+  text-shadow: none;
+  background-color: #fff;
+  color: $primary;
+}
+
+.course-info header.course-profile .intro-inner-wrapper {
+  box-shadow:none !important;
+}
+
+.content-wrapper .course-info header.course-profile {
+  height: auto;
+  padding-top: 0;
+}
+
+@media (max-width: 767px){
+  .course-info header.course-profile .intro-inner-wrapper div.table {
+    display: flex;
+    flex-direction: column-reverse;
+
+    .intro {
+      width: 100%;
       text-align: center;
-      width: flex-grid(12);
-
-      > .heading-group {
-        background: $white;
-        background: $course-header-bg;
-        border: 1px solid $border-color-3;
-        box-shadow: 0 4px 25px 0 rgba(0, 0, 0, 0.5);
-        padding: $baseline ($baseline*1.5);
-        position: relative;
-        z-index: 2;
-      }
-
-      &.main-search,
-      &.university-search {
-        text-align: center;
-
-        .heading-group {
-          display: inline-block;
-        }
-
-        .logo {
-          @include border-right(1px solid $light-gray);
-          @include margin-right(30px);
-          @include padding-right(30px);
-
-          display: inline-block;
-          height: 80px;
-          position: relative;
-          vertical-align: middle;
-
-          &::after {
-            @include right(0);
-
-            content: "";
-            display: block;
-            height: 80px;
-            position: absolute;
-            top: 0;
-          }
-
-          img {
-            height: 100%;
-          }
-        }
-
-        h1 {
-          color: $body-color;
-          font-family: $font-family-sans-serif;
-          font-style: italic;
-          font-weight: 700;
-          letter-spacing: 0;
-          text-transform: none;
-        }
-
-        h1,
-        h2 {
-          display: inline-block;
-          letter-spacing: 1px;
-          margin-bottom: 0;
-          text-shadow: 0 1px rgba(255, 255, 255, 0.8);
-          vertical-align: middle;
-        }
-      }
+    }
+
+    .pay-now-button {
+      margin: 0 auto;
+    }
+
+    .media {
+      width: fit-content;
+      margin: 0 auto;
+    }
+
+    img {
+      height: 160px;
+      max-width: 264px;
     }
   }
+
+  .course-info .container .course-sidebar {
+    width: 100%;
+  }
+}
+
+.wrapper-course-material .course-tabs {
+  padding: 0;
+}
+
+.tabs .tab a, .tabs .tab a:visited {
+  padding: 10px 15px 7px 15px;
+}
+
+body.view-in-course .wrapper-course-material .course-material .course-tabs {
+  padding: 0;
 }
 
-// +Search Input
-// ====================
-.find-courses {
-  .wrapper-search-context {
-    @include outer-container;
+div.wrapper-course-material .course-tabs .tab a,
+div.wrapper-course-material .course-tabs .tab a:visited {
+  padding: 10px 15px 7px 15px;
+}
 
-    @include rtl() { $layout-direction: "RTL"; }
```

Comment:
Why was RTL support removed? @luismorenolopera

Reply:
this affected course-container, but now we only handle an image without additional content, so that's why it's better not to add this